rfd/0000-rfds.md

```diff
@@ -0,0 +1,98 @@
+---
+authors: Grzegorz Uriasz (gorbak25@gmail.com)
+state: living-document
+---
+
+# RFD 0 - Request For Discussion
+
+## What
+
+Request For Discussion (RFD) is a design document format for non-trivial
+technical changes. It's also a process by which these documents are proposed, discussed, approved and tracked.
+
+## Why
+
+To invite anyone to propose new Hocus features, comment on existing design decisions and see what's upcoming. RFDs apply to major high level decisions, features, directions, and never to bugfixes, small changes, and refactors.
+
+Hocus as an **alpha software** allows RFDs to be highly lax to not slow down our 2 person team while opening the doors to any external feedback and accept proposals from the community. Expect some RFDs to be 2-3 sentences long at this stage.
+
+External Contributors aiming to introduce major changes or features are always required to open or amend an RFD before their contribution might get accepted.
```

Comment:
```suggestion
Hocus is still in **alpha** stage, so we keep RFDs flexible and simple to avoid slowing down our two-person team and to encourage external input and proposals from the community. Some RFDs may be very brief at this point.

External contributors who want to make significant changes or add new features must always open or update an RFD before their contribution can be accepted.
```